blog/views.py

```diff
@@ -0,0 +1,71 @@
+from blog.models import Post, Commentary, User
+from django.views import generic
+from django.urls import reverse_lazy, reverse
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpResponseRedirect
+
+
+class Index(generic.ListView):
+    model = Post
+    paginate_by = 5
+    template_name = "blog/index.html"
+    context_object_name = "post_list"
+    queryset = (Post.objects.all().select_related("owner")
```

Comment:
no need to add .all() method before select/perfetch related

Reply:
done